### PR TITLE
make usage compatible with YCSB and support table option

### PR DESCRIPTION
**Pull request by baiwfg2**

### CODE REVIEW

**Comment by baiwfg2** [comment]
Hey, what's your opinion about this patch ?

**Comment by disksing** [comment]
sorry I'm not very familiar with mongoDB. I'll see if I can find someone for help.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ log
 .idea
 .vscode
 vendor
+go-ycsb
```
Comment (iosmanthus) [inline .gitignore:8]: Execute `make` will output the binary to `bin/go-ycsb`, seems no need to ignore the root's

File: db/mongodb/db.go [hunk 2]
```diff
@@ -11,27 +11,23 @@ import (
...
-
-	mongodbUriDefault       = "mongodb://127.0.0.1:27017"
-	mongodbNamespaceDefault = "ycsb.ycsb"
-	mongodbAuthdbDefault    = "admin"
+	mongodbUrl        = "mongodb.url"
+	mongodbAuthdb     = "mongodb.authdb"
+	mongodbUsername   = "mongodb.username"
+	mongodbPassword   = "mongodb.password"
+
+	mongodbUrlDefault        = "mongodb://127.0.0.1:27017/ycsb?w=1"
```
Comment (iosmanthus) [inline db/mongodb/db.go:23]: We may add some comments here to inform where the default value comes from: https://github.com/brianfrankcooper/YCSB/tree/master/mongodb#mongodb-configuration-parameters

**Review by iosmanthus — COMMENTED** [review]
Rest LGTM.

**Comment by baiwfg2** [comment]
@iosmanthus Thanks. Done.

**Review by iosmanthus — APPROVED** [review]

**Comment by disksing** [comment]
Thanks @baiwfg2

**Pull request merged**